sdk/eventhub/event-hubs/src/eventHubConsumerClient.ts

```diff
@@ -165,20 +139,8 @@ export class EventHubConsumerClient {
    * @returns Promise<void>
    * @throws {Error} Thrown if the underlying connection encounters an error while closing.
    */
-  async close(): Promise<void> {
-    await this._client.close();
-    this._consumersMap.clear();
-  }
-
-  /**
-   * Provides the Event Hub runtime information.
-   * @param [options] The set of options to apply to the operation call.
-   * @returns A promise that resolves with EventHubProperties.
-   * @throws {Error} Thrown if the underlying connection has been closed, create a new EventHubClient.
-   * @throws {AbortError} Thrown if the operation is cancelled via the abortSignal.
-   */
-  async getProperties(options: GetPropertiesOptions = {}): Promise<EventHubProperties> {
```

Comment:
It might be useful to expose `getProperties` and `getPartitionProperties` on the consumer. The latter I think provides some information about the events in a partition that could be used to create an EventPosition.

Reply:
Done.